app/build.gradle

```diff
@@ -114,6 +114,9 @@ dependencies {
     //swipe_layout
     implementation 'com.daimajia.swipelayout:library:1.2.0@aar'
 
+    // SVG rendering library
+    implementation 'com.caverock:androidsvg-aar:1.4'
```

Comment:
Is it possible to fix the SVG issue without using this library?

Reply:
Hi @maskaravivek! I used it to decode the SVG image to get a `CloseableImage`.

Reply:
Seems like this how fresco has shown it in the official samples as well